pygmt/src/coast.py

```diff
@@ -73,10 +81,12 @@ def coast(self, **kwargs):
         parameter. Optionally, specify separate fills by appending
         **+l** for lakes or **+r** for river-lakes, and passing multiple
         strings in a list.
-    resolution : str
-        **f**\|\ **h**\|\ **i**\|\ **l**\|\ **c**.
-        Select the resolution of the data set to: (**f**\ )ull, (**h**\ )igh,
-        (**i**\ )ntermediate, (**l**\ )ow, and (**c**\ )rude.
+    resolution
```

Comment:
Copied from https://docs.generic-mapping-tools.org/dev/coast.html#d, but the +f modifier is not mentioned at all, since it's rarely used. It may not work as documented since in recent GMT versions, GMT can automatically download GSHHG and DCW datasets from the GMT data server.